plugins/org.eclipse.n4js.transpiler.es/src/org/eclipse/n4js/transpiler/es/transform/JSXTransformation.xtend

```diff
@@ -91,15 +95,21 @@ class JSXTransformation extends Transformation {
 		if(!inJSX) {
 			return; // this transformation is not applicable
 		}
+
+		// Transform JSXFragments and JSXElements	
+		val jsxFragments = collectNodes(state.im, JSXFragment, true);
 		val jsxElements = collectNodes(state.im, JSXElement, true);
-		if(jsxElements.empty) {
-			return; // nothing to transform
+		if (jsxFragments.isEmpty && jsxElements.isEmpty) {
```

Comment:
Also here: In case of a common superclass `JSXFragment` refactor if possible to reduce code duplication.